views/includes/documents/Userscripts-for-Safari.md

```diff
@@ -0,0 +1,34 @@
+## Userscripts for Safari
+
+<img src="https://is1-ssl.mzstatic.com/image/thumb/Purple126/v4/60/80/51/60805145-b1d7-ada7-774e-5baeb7055b15/AppIcon-85-220-4-2x.png/1024x0w.webp" width="128" height="128" align="right">
+
+Userscripts for Safari by Justin Wasack.
+
+### Installing Userscripts Extension
+
+To get userscripts going in Userscripts, first you have to install it from the [Apple Appstore][appstoreWebsite]. Install it like any other App.
+
+### Installing Userscripts
+
+Once Userscripts is installed, installing userscripts from [OpenUserJS.org][oujs] is simple. Navigate to the OpenUserJS page for the script, then click the blue "Install" button at the top of the page.
+You will be directed to a page showing the source code of that script.
+Open the Userscripts extension, and click where it says <kbd>Userscript Detected: <u>Click To Install</u></kbd>.
+
+<img width="413" alt="image" src="https://github.com/luckydonald-forks/OpenUserJS.org/assets/2737108/cb2565cb-e708-4e58-89d3-2e669dc9fabd">
```

Comment:
This will have to change to something repo local eventually... but good example for now.